frigate/app.py

```diff
@@ -397,6 +400,15 @@ def init_web_server(self) -> None:
             self.stats_emitter,
         )
 
+        self.fastapi_app = create_fastapi_app(
+            self.config,
+            self.detected_frames_processor,
+            self.storage_maintainer,
+            self.onvif_controller,
+            self.plus_api,
+            self.stats_emitter,
+        )
```

Comment:
@NickM-27 @hawkeye217 @blakeblackshear When you have some time please take a look. Please let me know what you think about this PR.

Reply:
will do, we are currently really busy with ffmpeg 7

Reply:
Got it, @NickM-27, totally understandable. I hope you guys get it working 💯 . 

I'll continue working on other endpoints in my free time. I already covered all **media** endpoints, and today I will convert **notification** and **preview** (at least).

So, in short:

* POC -> this PR (let's call it B1)
* media endpoints -> [This branch](https://github.com/iursevla/frigate-fork/tree/fastapi-poc-media-endpoints) (B2)
* notification and preview  -> on it (B3)

Basically dev <- B1 <- B2 <- B3 <- B.....

Reply:
I got to excited and converted it all on the third branch :D 

Here's the PR: https://github.com/blakeblackshear/frigate/pull/13871